WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewController.swift

```diff
@@ -6,6 +6,7 @@ final class ProductFormViewController: UIViewController {
 
     @IBOutlet private weak var tableView: UITableView!
 
+    private var originalProduct: Product
```

Comment:
I believe this deserves a comment about what "original" is and how we use it. Would you mind adding a comment? 🙂

Reply:
yup comments added here: https://github.com/woocommerce/woocommerce-ios/pull/1977/commits/ac142f80801aaf913befbf578998fdf78c2bed81